staged_dependencies.yaml

```diff
@@ -15,18 +15,9 @@ downstream_repos:
   insightsengineering/teal.transform:
     repo: insightsengineering/teal.transform
     host: https://github.com
```

Comment:
```suggestion
```
As this https://github.com/insightsengineering/teal.transform/issues/133 is going to decouple the scda from teal.trasform once merged . this should be removed